packages/viewer/src/components/viewer-pin-group/viewer-pin-group.css

```diff
@@ -1,16 +1,18 @@
 :host {
 
+  --viewer-annotations-pin-primary-color: var(--blue-700);
+  --viewer-annotations-pin-accent-color: var(--blue-200);
   /**
    * @prop  --viewer-annotations-pin-label-line-color: A CSS color that
    * specifies the color of the label line
    */
-   --viewer-annotations-pin-dot-color: var(--blue-600);
+   --viewer-annotations-pin-dot-color: var(--viewer-annotations-pin-primary-color);
```

Comment:
Not specific to this PR, but it is a bit confusing that the comment here refers to `--viewer-annotations-pin-label-line-color`, but the actual variable is `--viewer-annotations-pin-dot-color`. Could we update this comment to indicate what the `--viewer-annotations-pin-dot-color` variable does?